css/main.css

```diff
@@ -1,3 +1,248 @@
-/* Add your custom CSS here */
+.title-row {
+  justify-content: center;
+}
+.title-row p {
+  color: white;
+  font-size: 2rem;
+}
+.main-centre {
+  margin-top: 30px;
+  font-size: 1em;
+}
+.main-centre p {
+  text-align: center;
+}
+.about-section {
+  margin: 15px;
+}
+.row2 {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: 0px;
+  margin-left: 0px;
+}
+.about-col-6 {
+  position: relative;
+  min-height: 1px;
+  padding-right: 0px;
+  padding-left: 0px;
+}
+.about-col-6 h2 {
+  font-size: 80px;
+}
+.aboutus-text {
+  margin: 5rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+body {
+  font-size: 20px;
```

Comment:
Just so you know, the browser adds a default margin of 8px around the entire body. It is common practice to set the body to `margin: 0` to remove this.